src/librustdoc/html/render/mod.rs

```diff
@@ -1140,9 +1138,9 @@ themePicker.onblur = handleThemeButtonsBlur;
                     None
                 } else {
                     Some(Implementor {
-                        text: imp.inner_impl().print().to_string(),
+                        text: imp.inner_impl().print(cx.cache()).to_string(),
```

Comment:
Why `cx.cache()` here, then `&cx.cache` two lines down?

Reply:
It should be `cx.cache()` under as well.